tests/testthat/test-mergedatasetsbycase.R

```diff
@@ -115,28 +115,30 @@ test_that("Automatically determine match", {
 })
 
 test_that("Matching by variable labels", {
-    result <- MergeDataSetsByCase(data.set.names = c(findInstDirFile("cola1.sav"),
+    expect_warning(result <- MergeDataSetsByCase(data.set.names = c(findInstDirFile("cola1.sav"),
```

Comment:
It seems like this warning will appear a lot, where previously it did not appear. I think we shouldn't throw a warning for this, since it will already be highlighted in the output.